src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,8 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        String first = text.replaceAll("[^a-zA-Z0-9]", "").toLowerCase();
+        StringBuffer second = new StringBuffer(first).reverse();
```

Comment:
let's give  more informative names to variables. And it's better to use StringBuilder here cause it's faster

Reply:
done